### DISCUSSION TITLE
THEY HAVE FLAGGED MY ACCOUNT AND REMOVED FROM PUBLIC VISIBILITY

### DISCUSSION THREAD

**Discussion by kafishabbir** [community/community#147716] [New to GitHub] [answered]
THIS IS MY ACCOUNT https://github.com/kafiulshabbir
I have many important repositories there. We have published in journal and in the article I refer to the code there.

My reinstatement ticket is not being responded to for 1 months. **They are racist towards Russia and Russians.**

They did not have the basic decency to even explain the reason why account was flagged and removed from public visibility.

**Comment by ghost** [comment]
same for me https://github.com/RomThpt was flagged for no reason

**Reply by Tavrie** [reply]
> well maybe not "no reason" i did write the n word on my bio that i am a n-coder, and probably the algorithms detected my account as bad one, it was so embarrassing, in the middle of job interview -- i have to show them the code, they say they cannot open the link, they were like am i sure my repos are private or public, god damn it was embarrassing

i called someone a fag (I hope I don't get perm banned for saying fag and explaining to you) and why i said that? because i got mad because someone tried backdooring someone repository and that's how

**Reply by Tavrie** [reply]
here's my GitHub profile https://github.com/Zyverianox

**Reply by Tavrie** [reply]
> well maybe not "no reason" i did write the n word on my bio that i am a n-coder, and probably the algorithms detected my account as bad one, it was so embarrassing, in the middle of job interview -- i have to show them the code, they say they cannot open the link, they were like am i sure my repos are private or public, god damn it was embarrassing

and the flagged reason said for harassing or whatever I forgot

**Comment by Lut-Lat-Aung** [comment, marked as answer]
My account was suspended too. There is no warning or anything. I actually got this account back by trying this process
- Make reinstatement request with the account you got suspended. Make sure to add your suspended account username and gmail or email connected to it ( just in case ) . 

I got my account back within 3 weeks. They said it is because their abuse detecting systems highlight accounts that need to be manually reviewed.  I hope you get your account back.
If you need your account back urgently, 
In the statement request, say this. 

Title: GitHub Account Suspended Without Notice - Seeking Urgent Assistance

Hello GitHub Community,

My GitHub account (https://github.com/your-account) was suddenly suspended without any prior notice or explanation, which has severely impacted my work. I rely heavily on GitHub for my projects, and the suspension of my account means that my repositories are no longer accessible.
The Gmail associated is [youremail@gmail.com](mailto:youremail@gmail.com)
Here are the steps I've taken so far:

Submitted a Support Ticket: I submitted a support ticket through GitHub's support page.

My entire work has been suspended.
My repositories are inaccessible.
The lack of communication from GitHub has left me in the dark about the reason for the suspension.
Concerns:

There appears to be a pattern where many users experience sudden account suspensions without explanation.
It feels as though users on the free plan do not receive the same level of support as business users, which is disheartening.
Request:

I am urgently seeking assistance to understand the reason behind the suspension of my account.
I need help to restore access to my repositories as soon as possible.
Additional Context:

I have reviewed GitHub's terms of service and community guidelines to ensure compliance.
My work is critical to my projects and future creations, and this suspension has caused significant disruption.
I would greatly appreciate any advice, support, or guidance from the GitHub community or staff to resolve this issue promptly.

Thank you for your understanding and assistance.

Best regards,
Your name

**Reply by ghost** [reply]
Hi, I urgently need assistance, but it’s been a week since I submitted my first ticket ... But they are very slow ...

**Comment by ShiiiivanshSingh** [comment]
facing the exact same issue right now:(

**Reply by kafishabbir** [reply]
god bless you

**Comment by Tavrie** [comment]
> THIS IS MY ACCOUNT https://github.com/kafiulshabbir I have many important repositories there. We have published in journal and in the article I refer to the code there.
> 
> My reinstatement ticket is not being responded to for 1 months. **They are racist towards Russia and Russians.**
> 
> They did not have the basic decency to even explain the reason why account was flagged and removed from public visibility.

I think being racist can make your account flagged or flagged forever

**Comment by ghostinhershell** [comment]
Hi @kafishabbir and all 👋🏾 

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
